tests/kit/model_zoo/torchrec/__init__.py

```diff
@@ -1 +1 @@
-from .torchrec import *
+#from .torchrec import *
```

Comment:
Dont' comment out in your commit, only do this for your local testing.

Reply:
done

Reply:
It is still commented out.

Reply:
It's alright now, check it